configuration/device-tree.md

```diff
@@ -766,28 +831,30 @@ to get:
                 spi@7e2150C0 {
 ```
 
-<a name="part4.3"></a>
+The <a href="https://github.com/raspberrypi/utils">Utils</a> repo includes another DT utility - `ovmerge`. Unlike `dtmerge`, `ovmerge` combines file and applies overlays in source form. Because the overlay is never compiled, labels are preserved and the result is usually more readable. It also has a number of other tricks, such as the ability to list the order of file inclusion.
```

Comment:
Use [text](URL) syntax for offsite links?

Reply:
Ah, I needed to use `[test](url)`

Reply:
Yes, I got sucked in by the `<a` tags.